src/app/main.py

```diff
@@ -1,4 +1,4 @@
-from fastapi import FastAPI
+from fastapi import FastAPI, HTTPException
```

Comment:
I think we want to import `Dict` here for below response model
```suggestion
from typing import Dict

from fastapi import FastAPI, HTTPException
```